src/Spaces/spectralelement.jl

```diff
@@ -245,16 +245,15 @@ function SpectralElementSpace2D(
         Quadratures.quadrature_points(FT, quadrature_style)
     high_order_quad_points, high_order_quad_weights =
         Quadratures.quadrature_points(FT, high_order_quadrature_style)
+    tot_area = zero(FT)
```

Comment:
this doesn't appear to be used?

Reply:
Yes, indeed, I was already removing that. It was just there for my debugging purposes.